src/highlight.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+
+export const HighlightButton = () => {
+  return (
+    <button id="highlightBtn"></button>
+  );
+}
+
+export const startHighlight = ()  => {
+  console.log("here")
+    document.addEventListener("mouseup", () => {
+      console.log("mouseup --> GO");
+
+      const selection: Selection | null = document.getSelection();
+      if (selection == null || selection.rangeCount === 0) {
+        return;
+      }
+      
+      const range: Range = selection.getRangeAt(0);
+      let highlighted = range.extractContents();
+      
+      const node: Node | null = range.startContainer.parentNode;
+      if (node != null) {
+        let newNode = document.createElement("span");
+
+        newNode.appendChild(highlighted);
+        let selectedNode = (newNode as HTMLElement);
+        selectedNode.style.backgroundColor = "yellow";
```

Comment:
Please create a JIRA to change the class of the node to some class that identifies a user's annotation, so we can change colors easily. We won't do this for a while, so it's best to have a reminder.